swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
 class Item:
-    pass
\ No newline at end of file
+    
+    # constructor for class Item
+    def __init__(self, category = ""):
+        self.category = category
+        self.condition = 0
+        self.age = 0
```

Comment:
These two lines of code would make all instance of `Item` to start with a `condition` to of zero and `age` of zero. Instead, we should place optional parameters with values the initializer to default to if those values are not given: 
```
def __init__(self, category = "", condition = 0, age = 0):
    self.category = category
    self.condition = condition
    self.age = age
```